okta/webAuthnUserFactorProfile.go

```diff
@@ -0,0 +1,34 @@
+/*
+* Copyright 2018 - Present Okta, Inc.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*      http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+ */
+
+// AUTO-GENERATED!  DO NOT EDIT FILE DIRECTLY
+
+package okta
+
+import ()
+
+type WebAuthnUserFactorProfile struct {
+	AuthenticatorName string `json:"authenticatorName,omitempty"`
+	CredentialId      string `json:"credentialId,omitempty"`
+}
+
+func NewWebAuthnUserFactorProfile() *WebAuthnUserFactorProfile {
+	return &WebAuthnUserFactorProfile{}
+}
+
+func (a *WebAuthnUserFactorProfile) IsUserFactorInstance() bool {
+	return true
+}
```

Comment:
This is not a Factor.  It is a FactorProfile which I believe is not a UserFactorInstance.